--- origsrc/ConvertAll/source/helpview.py	2008-01-18 09:18:50.000000000 -0600
+++ src/ConvertAll/source/helpview.py	2008-07-15 00:38:49.593750000 -0500
@@ -31,7 +31,7 @@
         if sys.platform.startswith('win'):
             path = path.replace('\\', '/')
         self.textView.setSearchPaths([os.path.dirname(path)])
-        self.textView.setSource(QtCore.QUrl('file:///%s' % path))
+        self.textView.setSource(QtCore.QUrl('file://%s' % path))
         self.resize(520, 440)
         self.setWindowTitle(caption)
         tools = self.addToolBar('Tools')
